src/Entity/CronJob.php

```diff
@@ -108,6 +111,19 @@ public function setConfiguration($plugin_type, $configuration) {
   }
 
   /**
+   * {@inheritdoc}
+   */
+  public static function postDelete(EntityStorageInterface $storage, array $entities) {
+    foreach($entities as $entity) {
```

Comment:
all of this is a bit weird, we really are deleting here, and not reverting. Maybe we should deny delete access for system-provided cron hooks and just allow to disable them instead.